### PR TITLE
Switch to cheap-inline-source-maps for faster builds

### PR DESCRIPTION
**Pull request by balderdash**

### CODE REVIEW

**Comment by Bjvanminnen** [comment]
wonder if we should also update the other places we have inline-source-map in this file to use the cheap version

**Comment by balderdash** [comment]
Seems reasonable. I'll just make sure those configs still build with the cheap version.

**Comment by balderdash** [comment]
@Bjvanminnen PTAL

I'm using the `CHEAP` env variable to select `cheap-inline-source-map` I kept `inline-source-map` as the default, since it's a bit more correct.

File: apps/webpack.js [hunk 1]
```diff
@@ -85,8 +85,14 @@ if (envConstants.COVERAGE) {
   ];
 }
 
+var devtool = function () {
+  return process.env.CHEAP ?
+    'cheap-inline-source-map' :
+    'inline-source-map';
+};
```
Comment (Bjvanminnen) [inline apps/webpack.js]: does this need to be a function vs. just `var devtool = process.env.CHEAP ? 'cheap-inline-source-map' : 'inline-source-map';`

**Pull request merged**